### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by rdren0**
# Description

`Add tests to Components, some had to be skipped, some still need to be tested`

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other, Please explain:
`tests`

#Has This Been Tested?

- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`

# Checklist:

- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by MatthewKaplan** [comment]
AWESOME! More functionality will be added tomorrow so we can update the tests then.

File: package.json [hunk 1]
```diff
@@ -45,5 +45,12 @@
     "react-redux": "^7.0.3",
     "redux": "^4.0.1",
     "redux-devtools-extension": "^2.13.8"
-  }
+  },"jest": {
+    "collectCoverageFrom": [
```
Comment (letakeane) [inline package.json]: What's going on here?

File: src/Api/fetch/fetch.test.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+import { fetchAllNotes } from "./fetchAllNotes";
+import { fetchAddNote } from "./fetchAddNote";
+import { fetchDeleteNote } from "./fetchDeleteNote";
+import { fetchEditNote } from "./fetchEditNote";
+
+describe("fetch calls", () => {
+  let mockNotes;
+  let URL;
+
+  beforeEach(() => {
```
Comment (letakeane) [inline src/Api/fetch/fetch.test.js]: Sample comment

File: src/Api/fetch/fetch.test.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      mockGET = {
+        method: "GET",
+        body: JSON.stringify(),
+        headers: {
+          "Content-Type": "application/json"
+        }
+      };
+    });
+    it("should take an expected URL;", async () => {
+      await fetchAllNotes(mockGET);
```
Comment (letakeane) [inline src/Api/fetch/fetch.test.js]: More comments

**Pull request merged**